tests/integration/traces/traces_integ_base.py

```diff
@@ -53,11 +55,38 @@ def get_traces_command_list(
             command_list += ["--start-time", start_time]
         if end_time:
             command_list += ["--end-time", end_time]
-        if unformatted:
-            command_list += ["--unformatted"]
+        if output:
+            command_list += ["--output", output]
         if tail:
             command_list += ["--tail"]
         if beta_features:
             command_list += ["--beta-features"]
 
-        return command_list
\ No newline at end of file
+        return command_list
+
+    @staticmethod
+    def get_basic_deploy_command_list(
+        stack_name=None,
+        template_file=None,
+        capabilities=None,
+        resolve_s3=False,
+    ):
```

Comment:
I am in between using this one vs using the one in deploy tests. Even if we are using this one, I would recommend setting default values for some of these params. I believe `capabilities` and `resolve_s3` is always same, right?

Reply:
The one in deploy tests is an instance method and if we want to use it I need to inherit deploy base class. Even that I can't use the instance method in the class method `setUpClass()`. I am thinking that should me move all this kind of get_command_list method into a utility class since all the other test class may use this kind of get_command_list method.

Reply:
updated in the latest commit.